docker-compose.yml

```
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:10.3
+    env_file: env.d/db
+    ports:
+      - "5452:5432"
+
+  nextcloud:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      args:
+        - NEXTCLOUD_VERSION=${NEXTCLOUD_VERSION:-18.0.3-fpm}
+    image: openfun/nextcloud:${NEXTCLOUD_VERSION:-18.0.3-fpm}
```

I'd rather not name spacing the work-in-progress build to avoid conflicts between officially published images and local builds:

```suggestion
    image: nextcloud:${NEXTCLOUD_VERSION:-18.0.3-fpm}
```

---

No we can't do that because the image inherit from the official nextcloud image which has no namespace (`FROM nextcloud:18.0.3-fpm`)

---

and our namespace is `fundocker` not `openfun`

---

So I recommend using the `dev/` namespace for example, again to avoid confusion with official images.